packages/utils/LICENSE

```diff
@@ -0,0 +1,374 @@
+Mozilla Public License Version 2.0
```

Comment:
What's the point of including the license in each package? Can't we just place it in the root of the repo?

Reply:
Because we have to put LICENSE file to each npm package.